src/Metal/Defs.cs

```diff
@@ -226,6 +226,19 @@ public struct MTLDispatchThreadgroupsIndirectArguments {
 		public uint ThreadGroupsPerGrid3;
 	}
 
+	[Mac (10,14, onlyOn64: true), iOS (12,0), TV (12,0)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct MTLStageInRegionIndirectArguments {
+		public uint[] StageInOrigin;
+		public uint[] StageInSize;
```

Comment:
this is not correct and won't match the native representation
i.e. this is two pointers (to an unknown number of bytes), not 6x `uint_32`

Reply:
Add a unit test that check the (managed) size of the struct against the one you got from ObjC/sizeof